src/mbgl/util/utf.hpp

```diff
@@ -15,7 +15,28 @@ class utf8_to_utf32 {
         boost::u8_to_u32_iterator<std::string::const_iterator> end(utf8.end());
         return std::u32string(begin,end);
     }
+
```

Comment:
We should rewrite this whole file in terms of [`std::codecvt`](http://en.cppreference.com/w/cpp/locale/codecvt), dropping the boost dependency.